tests/common.py

```diff
@@ -1,15 +1,14 @@
 import contextlib
-import functools
 import itertools
 import os
 import unittest
+from collections.abc import Iterator
 from dataclasses import dataclass
 from importlib import resources
-from tempfile import NamedTemporaryFile, TemporaryDirectory
-from typing import Iterable, Optional
+from tempfile import NamedTemporaryFile, TemporaryDirectory, _TemporaryFileWrapper
```

Comment:
Is there a reason to import use private class rather than NamedTemporaryFile?

Reply:
This was to work around a mypy error:

> error: Function "tempfile.NamedTemporaryFile" is not valid as a type  [valid-type]

(relevant: https://stackoverflow.com/questions/64429113/how-should-a-namedtemporaryfile-be-annotated)

We don't run mypy on tests/ in CI, so could revert this or just add `# type: ignore`

Reply:
`# type: ignore` would be my preference. Just don't want to rely on private variables

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/995/commits/811f290ac54985e41fde4950201d01c96723a358